client/src/App.jsx

```diff
@@ -57,6 +58,20 @@ export default class App extends React.Component {
     }
 
 
+
+  onClickCancel(e){
```

Comment:
This function isn't needed because the onClickSignUp was refactored to work like Wesley's toggleModal function. The onClickSignup will revert back and forth to true and false when signup button and cancel button is clicked.

Reply:
I will do another merge with dev and accept your function.